app/services/research_safety_adapters/scishield_training_synchronizer.rb

```diff
@@ -80,6 +80,7 @@ def synchronize
     def api_unavailable?
       # Test API responses for 10 random users
       users.sample(10).map do |user|
+        sleep(batch_sleep_time)
```

Comment:
I'm fine with this, but waiting 20 sec between requests seems significantly more cautious than what we do in the `synchronize` method, right?  Do we need to use a smaller batch size?  or add a 1-2 sec wait between requests?